### PR TITLE
Better way to inject jupyter_primehub.py

### LINKED ISSUE #2: Better way to inject jupyter_primehub.py
Currently we need users to run helm with --set-file to inject jupyter_primehub.py.

See if subPath works for injecting jupyter_primehub.py.

Another option is to package the authenticator class

### PR DESCRIPTION
**Pull request by popcornylu**
Closes #2

Successful job
https://circleci.com/gh/InfuseAI/primehub/16

To verify
```
helm fetch infuseai/primehub --version=0.1-dcdafc4
tar -zxvf primehub-0.1-dcdafc4.tgz
cat primehub/values.yaml
```

### CODE REVIEW

File: helm/primehub/values.yaml [hunk 1]
```diff
@@ -44,6 +44,9 @@ jupyterhub:
       - containerPort: 9102
       - containerPort: 9125
         protocol: UDP
+    extraConfig:
+      primehub: |
+        ## jupyterhub_primehub.py
```
Comment (clkao) [inline helm/primehub/values.yaml]: Can we make the comment more verbose? such as saying the .py file is inlined while chart building, and for dev installing from directory, use --set-file

File: ci/update-values.py [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+        self.represented_objects[self.alias_key] = node
+    return node
+
+with open(valuefile, mode='r') as f:
+    result = yaml.load(f, Loader=yaml.FullLoader)
+
+with open(datafile, mode='r') as f:
+    data = f.read()
+
+result['jupyterhub']['hub']['extraConfig']['primehub'] = data
```
Comment (clkao) [inline ci/update-values.py]: please prepend some comment about the block being generated from another file, so people reading the rendered values.yaml know where to make actual changes

File: helm/primehub/values.yaml (hunk 1 above)
Comment (popcornylu) [reply helm/primehub/values.yaml]: Resolved

File: ci/update-values.py (hunk 2 above)
Comment (popcornylu) [reply ci/update-values.py]: Resolved

**Comment by aar0nTw** [comment]
ch843 https://app.clubhouse.io/infuseai/story/843/primehub-ce-better-way-to-inject-jupyter-primehub-py

**Comment by infuseai-dev** [comment]
LGTM

**Pull request merged**